ajax/libs/jquery.inputmask/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "jquery.inputmask",
   "version": "3.3.4",
   "description": "Inputmask is a javascript library which creates an input mask.  Inputmask can run against vanilla javascript, jQuery and jqlite.",
-  "filename": "jquery.inputmask.bundle.min.js",
+  "filename": "min/jquery.inputmask.bundle.min.js",
```

Comment:
`filename` change should follow file structure `version` instead of auto-update. Ref: https://github.com/cdnjs/cdnjs/pull/10727#discussion_r125230786

Reply:
@sufuf3 @extend1994 Hi!
How soon will make merge and release 4.0.4?

Reply:
@avradul , @sufuf3 , @extend1994,

This PR can be closed.
See https://github.com/cdnjs/cdnjs/pull/13242

Reply:
Thanks @RobinHerbots 👍